sites/fast-website/src/app/css/style.css

```diff
@@ -70,9 +89,100 @@ body {
 
 .footer {
   grid-column: 2;
+  display: flex;
+  align-items: center;
   margin-bottom: 84px;
 }
 
-.footer-navigation-item {
+.footer-brand svg {
+  width: 114px;
+  height: 40px;
+}
+
+.footer-navigation ul {
+  display: flex;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+.footer-navigation li {
   margin-inline-end: 50px;
 }
+
+@media screen and (max-width: 1330px) {
+  .wrapper {
+    grid-template-columns: minmax(5vw, 1fr) minmax(0px, 800px) minmax(5vw, 1fr);
+  }
+}
+
+@media screen and (max-width: 740px) {
+  .wrapper {
+    grid-template-columns: minmax(5vw, 1fr) minmax(0px, auto) minmax(5vw, 1fr);
+  }
+
+  .header {
+    grid-column: 1/4;
+  }
+
+  .header-spacer {
+    height: 94px;
+    width: 100%;
+    background-color: var(--background-color);
+    z-index: 3;
+  }
+
+  .header-brand {
+    position: absolute;
+    top: 28px;
+    left: 30px;
+    z-index: 4;
+  }
+
+  .header-menu {
+    display: block;
+    position: absolute;
+    top: 28px;
+    right: 20px;
+    z-index: 4;
+  }
+
+  .header-menu svg {
+    width: 35px;
+    height: 19px;
+  }
+  .header-github {
+    margin-left: 30px;
+  }
+
+  .header-navigation {
+    position: absolute;
+    top: 94px;
+    flex-direction: column;
+    align-items: flex-start;
+    justify-content: flex-start;
+    background-color: rgba(0, 0, 0, 0.95);
+    box-shadow: 0px 1.5px 1px rgba(0, 0, 0, 0.08), 0px 1.6px 5px rgba(0, 0, 0, 0.15);
+    transform: translateY(-100%);
+    transition: transform 0.1s ease-in;
+    z-index: 2;
+  }
+
+  .header-navigation ul {
+    flex-direction: column;
+    margin-top: 20px;
+  }
+
+  .header-navigation__open {
+    transform: translateY(0%);
+    height: 100%;
```

Comment:
`height` should always be set to 100%. Without it, the menu collapses right before transitioning out of view. Increase the `transition` duration in `.header-navigation` to investigate it more closely.

Reply:
height set